cfpb/consumerfinance.gov
# Remove nemo css and js

These styles are applicable to the contents of a `.nemo` class container on the retirement and paying for college apps, which I couldn't find on either.

The JS appears to be for a legacy nemo header that no longer exists.

## Removals

- Remove nemo css and js

## How to test this PR

1. `/frontend.sh` and run a local server
2. Inspect localhost:8000/consumer-tools/retirement/before-you-claim/, localhost:8000/paying-for-college/repay-student-debt/, localhost:8000/paying-for-college/your-financial-path-to-graduation/, and the paying-for-college2 URL (see @higs4281 for URL)

…and compare to production. None of those pages should have a `nemo` class or JS errors.
